### PR TITLE
Update search change log release notes

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

File: releases/2020-05/java.md [hunk 1]
```diff
@@ -172,13 +172,17 @@ If you have a bug or feature request for one of the libraries, please post an is
...
-
-### New Features [Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md#700-beta2-2020-05-07)
-
+### Service Bus ([Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md#700-beta2-2020-05-07))
+Here are some of the highlights:
+#### New Features 
 - Allow receiving from first available single and multiple sessions through ServiceBusReceiverAsyncClient.
 - Add send overload that accepts an Iterable of ServiceBusMessage.
 
+### Search ([Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/search/azure-search-documents/CHANGELOG.md#100-beta3-2020-05-05))
```
Comment (alzimmermsft) [inline releases/2020-05/java.md:181]: Should we call this `Search Documents`?

File: releases/2020-05/java.md [hunk 2]
```diff
@@ -172,13 +172,17 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+Here are some of the highlights:
+- Changed Azure Search service version from `2019-05-06` to `2019-05-06-Preview`
+- Added helper class `FieldBuilder` which converts a strongly-typed model class to `List<Field>`.
```
Comment (alzimmermsft) [inline releases/2020-05/java.md:184]: Should these be under a `#### New Features` header? Also should there be a header for the breaking changes made (though I don't know if we are doing that for previews @czubair).

File: releases/2020-05/java.md (hunk 1 above)
Comment (sima-zhu) [reply releases/2020-05/java.md:181]: .dotnet is using Search.
https://github.com/Azure/azure-sdk/blob/master/releases/2020-05/dotnet.md#search-changelog

File: releases/2020-05/java.md (hunk 2 above)
Comment (sima-zhu) [reply releases/2020-05/java.md:184]: I can add this to keep consistent

**Pull request merged**